ViAn/GUI/mainwindow.cpp

```diff
@@ -232,25 +237,55 @@ void MainWindow::resizeEvent(QResizeEvent* event) {
  * @param newPos current position of the slider
  */
 void MainWindow::on_videoSlider_valueChanged(int newPos){
```

Comment:
Refactor this to make it easier to read. If the method does more than one thing, separate based on those.